contents/docs/experiments/funnels-statistics.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Statistical methodology for funnel experiments
+---
+
+Funnel experiments use Bayesian statistics with a beta model to evaluate the **win probabilities** and **credible intervals** for an experiment. [Read the statistics primer for an overview](/docs/experiments/statistics-primer) if you haven't already.
+
+## What the heck is a Beta model?
```

Comment:
I'd probably tone down the casual tone here :D Though I generally appreciate it, Experiment have a high standard for rigor, so I feel a more formal language better communicates this.